File: GGAndroid_2.0/GGDemo/src/main/java/io/ganguo/demo/view/activity/SelectManagerDemoActivity.java
package io.ganguo.demo.view.activity;

import android.content.Context;
import android.content.Intent;

import io.ganguo.demo.view.activity.base.GGHFRVModelActivity;
import io.ganguo.demo.viewmodel.activity.SelectManagerDemoVModel;
import io.ganguo.vmodel.ViewModelActivity;
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding;

/**
 * SelectManager Demo
 * Created by Roger on 12/05/2017.
 */

public class SelectManagerDemoActivity extends GGHFRVModelActivity<IncludeHfRecyclerBinding, SelectManagerDemoVModel> {

    public static Intent intentFor(Context context) {
        Intent i = new Intent(context, SelectManagerDemoActivity.class);
        return i;
    }

    @Override
    public SelectManagerDemoVModel createViewModel() {
        return new SelectManagerDemoVModel();
    }

    @Override
    public void onViewAttached(SelectManagerDemoVModel viewModel) {
        super.onViewAttached(viewModel);
    }
}

File: GGAndroid_3.0/main/app-mvvm-v1-sample/src/main/java/io/ganguo/demo/view/activity/SelectManagerDemoActivity.kt
package io.ganguo.demo.view.activity

import android.content.Context
import android.content.Intent

import io.ganguo.demo.view.activity.base.GGHFRVModelActivity
import io.ganguo.demo.viewmodel.activity.SelectManagerDemoVModel
import io.ganguo.viewmodel.core.BaseViewModel
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding

/**
 * SelectManager Demo
 * Created by Roger on 12/05/2017.
 */

class SelectManagerDemoActivity : GGHFRVModelActivity<IncludeHfRecyclerBinding, SelectManagerDemoVModel>() {

    override fun createViewModel(): SelectManagerDemoVModel {
        return SelectManagerDemoVModel()
    }

    override fun onViewAttached(viewModel: SelectManagerDemoVModel) {
    }

    companion object {

        fun intentFor(context: Context): Intent {
            return Intent(context, SelectManagerDemoActivity::class.java)
        }
    }

}
